CMakeLists.txt

```diff
@@ -38,8 +38,14 @@ if (NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE "Release")
 endif()
 
+option(USE_RELATIVE_LIB_PATH "Use relative path to libear")
+
 set(EAR_LIB_FILE ${CMAKE_SHARED_LIBRARY_PREFIX}ear${CMAKE_SHARED_LIBRARY_SUFFIX})
-set(DEFAULT_PRELOAD_FILE ${CMAKE_INSTALL_FULL_LIBDIR}/${EAR_LIB_FILE})
+if (USE_RELATIVE_LIB_PATH)
```

Comment:
(sorry for the late reply)

I guess there is a risk that the user has already specified a special LD_LIBRARY_PATH to override some libs, and then by adding wherever libear is istalled to that may have confusing results.

The build environment where I'm trying to get Bear working does lots of tricks moving stuff around and pointing out paths with LD_LIBRARY_PATH.  But just having the "Allow overriding libear path via environment" commit will cover my needs. So I can drop this patch if there is risk of problems with multilib.